Sprint-3/alarmclock/alarmclock.js

```diff
@@ -1,4 +1,58 @@
-function setAlarm() {}
```

Comment:
Hi @sajadsoorni  can you please add your HTML?

Reply:
Hi @ellietms, 
I’ve added the HTML along with the latest updates. Please let me know if anything else needs to be adjusted.